rust/relay/ebpf-turn-router/Cargo.toml

```diff
@@ -4,6 +4,11 @@ version = "0.1.0"
 edition = { workspace = true }
 license = { workspace = true }
 
+[package.metadata.cargo-udeps.ignore]
+normal = ["aya-ebpf", "aya-log-ebpf", "ebpf-shared", "network-types"]
+build = ["which"]
+development = ["hex-literal", "ip-packet"]
```

Comment:
Instead of ignoring them here, it should be possible to make them target dependent. Can be done in a follow-up though so we can merge this soon.

Reply:
I just dropped the change for Windows, as I can't really test it here and it wasn't really important here - I'll make a follow-up to make them target dependent, and see how it fares with Windows, thanks for the suggestion!